src/uaf_hash.c

```diff
@@ -18,19 +18,12 @@
 #define _uaf_hash_
 #include "uaf_encode.c"
 
-#ifdef VMS
-#include <ssdef.h>
-#define SSs_ABORT SS$_ABORT
-#define SSs_BADPARAM SS$_BADPARAM
-#define SSs_NORMAL SS$_NORMAL
-#else
```

Comment:
This must not be removed! And it doesn't affect AIX.

Reply:
Sorry, it was late yesterday.

Reply:
This was my doing. I will restore these definitions.